clifford.html

```diff
@@ -0,0 +1,97 @@
+<body>
```

Comment:
Should there be an `examples` folder to contain all of these separate files (e.g. `vietnam.html`)?

Reply:
I have to think this through. I'm considering losing the current `index.html` and replacing it with a more fleshed-out `clifford.html` with selectors and stuff, because I don't like that you have to install a python module just to get things up and running. It's possible that some of this actually should be living in tests. (And `vietnam.html` shouldn't even have been checked in... I must have done that for my own convenience while swapping machines at some point and forgot to delete.)